### DISCUSSION TITLE
Change claim block from Chest to special block?

### DISCUSSION THREAD

**Discussion by lostINParadies** [GriefPrevention/GriefPrevention#763] [Ideas]
Hey, we creating now a adventure server and need an plugin like this. Is it possible to change the claim block from chest to an other special block like a special block you only can buy? so you place the spcial block on the ground and you claim the land.

**Comment by bigpresh** [comment]
I'm not sure what you mean by "the claim block".

Generally, you create claims via the claim tool (default is a gold shovel) or the /createclaim command.

By default, a small claim is automatically created around the first chest a player places, but that only happens the first time.

Assuming what you're after is a limit on claims people can create, using up an entitlement each time they create a claim, I can't think of a good way to do that other than via an addon of some sort.

GriefPrevention itself lets you control how many claim blocks (i.e. total area) a player can have, but it's more or less up to them whether they use those blocks as one large claim or several smaller ones.

**Comment by lostINParadies** [comment]
I mean. You create your first region with a chest. I would like to replace the chest with another block. For example, that you create your first region with a banner.

![2020-03-29_19 48 05](https://user-images.githubusercontent.com/19432313/77856343-c2a97380-71f6-11ea-8de4-5f8f275e92f9.png)
![2020-03-29_19 48 20](https://user-images.githubusercontent.com/19432313/77856344-c63cfa80-71f6-11ea-88cd-e34957d03c86.png)
![2020-03-29_19 49 33](https://user-images.githubusercontent.com/19432313/77856345-c76e2780-71f6-11ea-9f14-971db554b736.png)

**Comment by bigpresh** [comment]
The fact that the first chest you place is hard-coded, so no, that wouldn't work.

I'm not quite sure the value of being able to change that, anyway; a player could just craft a golden shovel and create a claim manually rather than having a banner.

An addon could apply whatever rules you want, though - but I don't personally see  that as a feature that belongs in GP's core.
